docs-ref-conceptual/format-output-azure-cli.md

```diff
@@ -255,11 +266,27 @@ Console output showing the new password in the console.
 
 A better solution is to return sensitive information to a variable.
 
+# [Bash](#tab/bash)
+
 ```azurecli-interactive
+# Bash Script
 # reset service principal credentials returning results to a variable
 myNewPassword=$(az ad sp credential reset --id myServicePrincipalID --query password --output tsv)
+
+# Display the new password (remove this line in production for security)
+echo "New password: $myNewPassword"
 ```
+# [PowerShell](#tab/powershell)
+
+```azurecli
+# PowerShell script
+# Reset service principal credentials and capture the new password
+$myNewPassword = (az ad sp credential reset --id myServicePrincipalID --query password --output tsv)
 
+# Display the new password (remove this line in production for security)
+Write-Host "New password: $myNewPassword"
```

Comment:
```suggestion
echo "New password: $myNewPassword"
```

Reply:
@v-abdriaz , `write-host` is correct but it doesn't work in Bash.  `echo` works in both environments.